packages/comms/package.json

```diff
@@ -69,10 +69,10 @@
         "tmp": "0.2.1"
     },
     "devDependencies": {
-        "@hpcc-js/bundle": "^2.11.3",
+        "@hpcc-js/bundle": "^2.10.23",
```

Comment:
When I was building this to test, this version number showed as incorrect; there doesn't seem to be a 2.11.3 for bundle